server/build/babel/preset.js

```diff
@@ -7,10 +7,21 @@ const productionPlugins = isProduction ? [
   require.resolve('babel-plugin-transform-react-remove-prop-types')
 ] : []
 
+const es2015Config = {}
+
+// Do not transform ES2015 modules if we are inside Next.js
+// That's because webpack2 knows how to handle it.
+// (And that's how it can do code splitting)
+//
+// But in other environements like Jest, we should transform it.
+if (process.env.INSIDE_NEXT) {
```

Comment:
This might be not the best solution. But this is the best option I could find out.

I tried using passing an option to our babel config. But, when the user adds a custom `.babelrc`, he/she has to add that option manually. Which is not nice.